docs/UserMenu.md

```diff
@@ -0,0 +1,108 @@
+# User Menu
+
+The `<pxb-user-menu>` is an Avatar that opens a Menu when clicked. It is typically used in the top-right corner of an application and indicates who is logged in.
+
+<div style="align-items: center; display:flex; justify-content: space-around">
+
+<img width="30%" alt="UserMenu Avatar" src="./images/userMenuAvatar.png">
+<img width="35%" alt="UserMenu Opened" src="./images/userMenuOpened.png">
+
+</div>
+
+The Menu header can be constructed by supplying a `menuTitle` and optional `menuSubtitle`, or your own custom content. The Menu body is entirely custom content.  See the API section below for more details.  
+
+## Usage
+
+<div style="width: 100%; text-align: center">
+    <img width="100%" style="max-width: 500px" alt="User Menu Anatomy" src="./images/userMenuAnatomy.png">
+</div>
+
+```typescript
+// app.module.ts
+import { UserMenuModule, InfoListItemModule } from '@pxblue/angular-components';
+...
+imports: [
+    UserMenuModule,
+    InfoListItemModule
+],
+...
+```
+
+```tsx
+let open = false;
+const items = [
+    {
+        title: 'Account',
+        icon: 'settings',
+    },
+    {
+        title: 'Contact Us',
+        icon: 'mail',
+    },
+    {
+        title: 'Log Out',
+        icon: 'logout',
+    }
+];
+```
+
+```html
+<pxb-user-menu avatarValue="HA" menuTitle='Sample Title' [(open)]="open">
+    <mat-nav-list pxb-body>
+        <pxb-info-list-item *ngFor="let item of items" [dense]="true" (click)="open=false">
+            <mat-icon pxb-icon>{{item.icon}}</mat-icon>
+            <div pxb-title>{{item.title}}</div>
+        </pxb-info-list-item>
+    </mat-nav-list>
+</pxb-user-menu> 
+```
+
+## API
+
+Parent element (`<pxb-user-menu>`) attributes:
+
+<div style="overflow: auto;">
+
+| @Input       | Description                                      | Type                        | Required | Default                                                                                                                     |
+| ------------ | ------------------------------------------------ | --------------------------- | -------- | -------                                                                                                                     |
```

Comment:
Can we clean up these tables a little bit to make the column separators align? I wish there was a VSCode plugin for this

Reply:
Should be all  cleaned up now

Reply:
> I wish there was a VSCode plugin for this

If we `prettier` the UserMenu.md, I think it automatically does it for you. I have our prettier hooked up with VSC on save to satisfy my OCD